lib/ansible/plugins/connection/ssh.py

```diff
@@ -628,7 +628,10 @@ def _file_transport_command(self, in_path, out_path, sftp_action):
                 cmd = self._build_command('sftp', to_bytes(host))
                 in_data = u"{0} {1} {2}\n".format(sftp_action, shlex_quote(in_path), shlex_quote(out_path))
             elif method == 'scp':
-                cmd = self._build_command('scp', in_path, u'{0}:{1}'.format(host, shlex_quote(out_path)))
+                if sftp_action == 'get':
+                    cmd = self._build_command('scp', u'{0}:{1}'.format(host, shlex_quote(out_path)), in_path)
```

Comment:
is this right? src= in fetch module is still the path for the file to be copied. eg:

fetch: src=/etc/resolv.conf dest=/tmp

would become

scp {{remote_hostname}}:/etc/resolv.conf /tmp

So the correct translation would be:

cmd = self._build_command('scp', u'{0}:{1}'.format(host, shlex_quote(in_path)), out_path)

// Anderson